src/get-started/quickstart-py.md

```diff
@@ -68,6 +68,24 @@ Install the latest Python SDK from PyPI:
 pip install apache-beam
 ```
 
+#### Extra Requirements
```

Comment:
How about something like this at the end?

(It is possible to use multiple features by `pip install apache-beam[feature1,feature2,...]`.)